### PR TITLE
update SMTP

### PR DESCRIPTION
**Pull request by tmfrnz**
related to #105

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ env:
 
 jobs:
   build-and-push-image:
-    if: ${{ github.ref == 'refs/heads/master' }}
+    if: ${{ github.ref == 'refs/heads/master' || github.ref == 'refs/heads/smtp-from' }}
```
Comment (parndt) [inline .github/workflows/deploy.yml:16]: @tmfrnz if you want we can remove this check and allow push from any branch?

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -77,6 +77,30 @@ jobs:
...
+        run: bin/kamal env push
+        env:
+          RAILS_SERVE_STATIC_FILES: true
+          RUBY_YJIT_ENABLE: 1
+          DEPLOY_LETSENCRYPT_EMAIL: ${{ secrets.DEPLOY_LETSENCRYPT_EMAIL }}
+          DEPLOY_REGISTRY_PASSWORD: ${{ secrets.GITHUB_TOKEN }}
+          DEPLOY_REGISTRY: ${{ env.REGISTRY }}
+          DEPLOY_SSH_HOST: ${{ secrets.DEPLOY_SSH_HOST }}
+          DEPLOY_USERNAME: ${{ github.actor }}
+          DEPLOY_WEB_HOST: ${{ secrets.DEPLOY_WEB_HOST }}
```
Comment (parndt) [inline .github/workflows/deploy.yml:90]: I'm not 100% sure that these `DEPLOY_` ones need to be pushed 🤔

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (tmfrnz) [reply .github/workflows/deploy.yml:16]: done =)

File: .github/workflows/deploy.yml (hunk 2 above)
Comment (tmfrnz) [reply .github/workflows/deploy.yml]: removed - thanks

**Pull request merged**